src/lib/updating-element.ts

```diff
@@ -462,7 +472,25 @@ export abstract class UpdatingElement extends HTMLElement {
    * @returns {Element|DocumentFragment} Returns a node into which to render.
    */
   protected createRenderRoot(): Element|ShadowRoot {
-    return this.attachShadow({mode : 'open'});
+    const shadowRoot = this.attachShadow({mode : 'open'});
+    const styles = (this.constructor as typeof UpdatingElement).styles;
```

Comment:
I (weakly) feel like all this should be in LitElement, since it does the actual rendering.

Reply:
It seems like the `shadowRoot` rendering (creation + styles) should go together, but I think it's reasonable that this go into `LitElement` and not `UpdatingElement`. Let's discuss separately.